content/en/docs/concepts/services-networking/gateway.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Gateway
+content_type: concept
+description: >-
+  Make network services available by using an extensible, role-oriented, protocol-aware
+  API. Gateway is a group of resources that provide dynamic infrastructure provisioning
+  and advanced traffic routing for Kubernetes services.
+weight: 55
+---
+
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.29" state="stable" >}}
+
+Make network services available by using an extensible, role-oriented, protocol-aware configuration
+mechanism. The [Gateway][gw-api] API consists of [resources][resources] that provide dynamic
+infrastructure provisioning and advanced traffic routing.
+
+The Gateway project is maintained by the [SIG-NETWORK][sig-net] community outside of Kubernetes.
+Since certain Gateway API resources are considered stable, a subset of the full Gateway [documentation][gw-api]
+is included here.
```

Comment:
I wouldn't state this. We should start to list all the Gateway API kinds in this document, even if just as stubs and hyperlinks. That's what our readers would want, especially given that the core API kinds are GA.

Reply:
@robscott if we take ^ approach, then we need to use `{{< feature-state for_gwapi_version="v1.0" state="stable|beta|alpha" >}}` for each.

Reply:
Hmm; that shortcode (or a revised version of it) doesn't feel like the right fit to me.

Reply:
@sftim I created a shortcode for Gateway API features. Please take a look and let me know if you still prefer omitting it from this PR.